tests/cursor.py

```diff
@@ -74,6 +81,31 @@ def random_cursor(mocker, random_response_dict):
     return Cursor(client, "campaign", representations.raw)
 
 
+@pytest.fixture
+def ordered_cursor(mocker, ordered_response_dict):
+    client = AppNexusClient("test", "test")
+    mocker.patch.object(client, "get")
+    client.get.side_effect = ordered_response_dict
+    return Cursor(client, "campaign", representations.raw)
+
+
+def mock_ordered_cursor(mocker, start=0, count=COLLECTION_SIZE, factor=1):
+    client = AppNexusClient("test", "test")
+    mocker.patch.object(client, "get")
+    client.get.side_effect = gen_ordered_collection(start, count) * factor
+    cursor = Cursor(client, "campaign", representations.raw)
+    mocker.patch.object(cursor, "get_page", wraps=cursor.get_page)
+    return cursor
+
+
+@pytest.fixture
+def double_ordered_cursor(mocker, ordered_response_dict):
+    client = AppNexusClient("test", "test")
+    mocker.patch.object(client, "get")
+    client.get.side_effect = ordered_response_dict * 2
+    return Cursor(client, "campaign", representations.raw)
```

Comment:
This fixture doesn't seem used anywhere.

Reply:
Removed